### PR TITLE
Blood Cult Nerf: Restricted Nar-Sie summoning locations

### PR DESCRIPTION
**Pull request by Robustin**
Blood cult is our most balanced mode by winrate, but as I continue to delivery QOL improvements, updates, ease of use, and flat out buffs - then some added difficulty is required as well. 

This PR selects 3 rooms at roundstart and informs the cult that the summoning ritual must be done in one of those 3. These rooms are all on station-level and are all "rooms" in the traditional sense, maintenance and other mini-rooms in maint are not included. The cult will have several forms of feedback to remind them of this requirement. RNG will play a role of course, sometimes the cult will get virology or atmospherics, but more often they will get three locations that are less than ideal - where strategy and forethought will be necessary for success. 

:cl: Robustin
balance: The blood cult can only attempt to summon Nar-Sie in one of three rooms that are randomly selected at round-start. 
/:cl:

### CODE REVIEW

**Comment by Supermichael777** [comment]
no

**Comment by Jalleo** [comment]
YES

File: code/datums/antagonists/datum_cult.dm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#define SUMMON_POSSIBILITIES 3
```
Comment (Jalleo) [inline code/datums/antagonists/datum_cult.dm]: Any chance this can be a config instead?

**Comment by Iamgoofball** [comment]
Can we apply this to clockcult too

**Comment by Balcore** [comment]
This gets a 👍

Also what @goof said. 

Would be interesting to see Clock cult have to shore up summoning areas in advance with their base building abilities.

File: code/datums/antagonists/datum_cult.dm (hunk 1 above)
Comment (Robustin) [inline code/datums/antagonists/datum_cult.dm]: Anything is possible but my experience is nobody ever touches these kind of config settings

**Comment by Supermichael777** [comment]
SUMMON NARSERIE IN THE CHAPEL ARMORY OR PERMA BRIG

**Comment by Robustin** [comment]
Permabrig is a great one you nerd. Bust in from space, drop a teleport rune/summon everyone and two barrier runes later you've got one of the best spots to summon in.

The chapel isn't a horrible spot either, there's often only 3 tiles to defend and its wide open nature makes it ideal for a blood boil trap.

**Comment by DoctoreMad** [comment]
>Blood cult is our most balanced mode by winrate, but as I continue to delivery QOL improvements, updates, ease of use, **and flat out buffs - then some added difficulty is required as well.**

Please rewrite OP into

>Blood cult is our most balanced mode by winrate, but as I continue to fuck with it for no fucking reason with the chance I might unbalance it

Because holy shit that is AWFUL, why would you "buff and nerf" something that by your own word doesn't need a nerf? I get the QoL updates and shit but the last part is unforgivable.

**Comment by Iamgoofball** [comment]
did you read what he said?

**Comment by Robustin** [comment]
Because too much of cult's power is focused into a couple runes/talismans. Buffing the weaker stuff makes a better mode but also increases the antag's power, so nerfs to their really-strong-but-not-fun tactics is a great way to reign in power.

**Comment by Magikarpet** [comment]
what would happen if someone intentionally bombs out the entire area and basically removes it from the map

would it prevent summoning

**Comment by calzilla1** [comment]
Do it for cuck cult and THEN I'll :+1: it

**Comment by Bawhoppen** [comment]
QoL balance babies :-1:

**Comment by Bawhoppen** [comment]
actually wait if it's random then I retract my :-1: and replace with a :confused:

**Comment by Robustin** [comment]
@Magikarpet No, if you replace the floor you could still summon in that room. Alternatively you can just use the other 2 rooms.

**Comment by BeeSting12** [comment]
I don't see a problem with this, at very least worth a try

**Comment by BeeSting12** [comment]
>thumbs upping your own PR

**Comment by Robustin** [comment]
@Beesting12 

Ah you're right I forgot a heart and confetti

**Comment by PKPenguin321** [comment]
Poorly named areas on maps will fuck this up, as well as rooms that can't fit a 3x3 rune

**Comment by Robustin** [comment]
Most of the rooms that qualify for summon locations aren't the weird ones, and having a set of 3 means if one location is too ambiguous the others should be more obvious. Juggernauts/Artificers can fix any issues with tight spaces too.

**Pull request merged**